geanyctags/README

```diff
@@ -109,9 +111,10 @@ Get the code from::
 Ideas, questions, patches and bug reports
 =========================================
 
-Please direct all questions, bug reports and patches to the plugin author using the
-email address listed below or to the Geany mailing list to get some help from other
-Geany users.
+Please direct all questions, bug reports and patches to the combined geany-plugins
+project at https://github.com/geany/geany-plugins and open the corresponding
+bug report or pull request there. To notify the author of this plugin about
+your post, mention him using his github user name (@techee).
```

Comment:
Maybe mention that the github username can usually be found in the MAINTAINERS file.

Reply:
I'll probably skip this level of indirection and mention it directly in the documentation. I normally check geany-plugins only from time to time but I have an email notification enabled when someone mentions me.